curriculum/js1/4.js

```diff
@@ -7,7 +7,11 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+  let str = "";
+  for(let i=0; i<a; i++){
```

Comment:
awesome.... this looks great can you try doing this with recursion instead of loops....

Reply:
Thank you for a reminder. I thought that this exercise is easier done with a for-loop. I used recursion in an exercise with callback function repetition because I didn't need to create a nested function in solution().

Reply:
You want to get to the point where coding recursively is easier and cleaner: 

```
solution = (a, b, result="") => {
  if( !a ) return result;
  return solution(a-1, b, result + b);
}
```